src/public/modules/core/services/gtag.client.service.js

```diff
@@ -1,10 +1,18 @@
+const get = require('lodash/get')
+
 angular.module('core').factory('GTag', ['$rootScope', '$window', GTag])
 
 function GTag($rootScope, $window) {
   // Google Analytics tracking ID provided on signup.
   const GATrackingID = $window.GATrackingID
   let gtagService = {}
 
+  const userEmail = () => {
```

Comment:
tiniest of nits but we normally try to name functions with verbs to differentiate them from variables, so something like `getUserEmail()` instead of `userEmail()` would be clearer! Functions which return boolean values are exceptions, since we often name them starting with `has` or `is`.

Reply:
fair point! made the edits.